app/src/main/java/com/youssef/fixit/Jobs/JobsView.java

```diff
@@ -3,8 +3,8 @@
 import com.youssef.fixit.Models.Jobs.Jobs;
 
 interface JobsView {
-    void OnGetJobs(Jobs jobs);
+    void displayJobsData(Jobs jobs);
```

Comment:
I don't know about the backend side and how the data is retrieved but always try to create a single entity so if the backend return list of jobs keep your entity singular `Job` 
And you can receive List<Job> 
Other data inside `Jobs` entity is more related to the network layer and the view shouldn't be considered with all this extra information or even be able to see all this.